sdk/core/core-auth/src/tokenCredential.ts

```diff
@@ -74,6 +74,11 @@ export interface AccessToken {
    * The access token's expiration timestamp in milliseconds, UNIX epoch time.
    */
   expiresOnTimestamp: number;
+
+  /**
+   * The timestamp when the access token should be refreshed, in milliseconds, UNIX epoch time.
+   */
+  refreshesOnTimestamp?: number;
```

Comment:
nit: should this be `refreshAfterTimestamp`?

Maybe not worth changing since I like that the current name mirrors the wire name.

Reply:
I am not sure, my brain has forgotten the correct usage of prepositions, only biased towards the field names. I am okay with anything.

Reply:
Updated to `refreshAfterTimestamp`